tests/parsers/test.manifestjson.js

```diff
@@ -279,3 +279,26 @@ describe('ManifestJSONParser update_url', function() {
     assert.equal(manifestJSONParser.isValid, true);
   });
 });
+
+
+describe('ManifestJSONParser with comments', function() {
+
+  it('parses JSON', () => {
+    var addonLinter = new Linter({_: ['bar']});
+    var json = `// I am a JSON comment, sigh\n${validManifestJSON()}`;
+    var manifestJSONParser = new ManifestJSONParser(json,
+                                                    addonLinter.collector);
+    assert.equal(manifestJSONParser.isValid, true);
+  });
+
+  it('returns the correct error for malformed JSON', () => {
+    var addonLinter = new Linter({_: ['bar']});
+    var json = `{"something": true,\n// I am a JSON comment, sigh\nblah}`;
+    var manifestJSONParser = new ManifestJSONParser(json,
+                                                    addonLinter.collector);
+    assert.equal(manifestJSONParser.isValid, false);
+    var errors = addonLinter.collector.errors;
+    assert.equal(errors[0].code, messages.MANIFEST_JSON_INVALID.code);
+    assert.include(errors[0].message, 'Invalid JSON in manifest file.');
+  });
+});
```

Comment:
Since we are "cleaning" this with a fully capable ECMAScript parser I would love to see examples of trying to inject some JavaScript code in there, e.g do some `var bla = "foo"` and make sure they don't appear in the final output.

Also, an actual valid example like the docs from https://developer.chrome.com/extensions/manifest (I just assume they're very valid) would be nice

```javascript
{
  // Required
  "manifest_version": 2,
  "name": "My Extension",
  // Make the hell sure to use semvar.org if increasing this
  "version": "0.0.1"
}
```

…would be something I'd do, maybe.